summaries/style_gan3.md

```diff
@@ -0,0 +1,122 @@
+# STYLEGAN 3 : Alias-Free Generative Adversarial Networks
+Tero Karras, Miika Aittala ,Samuli Laine , Erik Härkönen, Janne Hellsten, Jaakko Lehtinen, Timo Aila 
+
+## Summary
+
+* The typical GANs depends on absolute pixel coordinates in an unhealthy manner. This manifests itself as, e.g., detail appearing to be glued to image coordinates instead of the surfaces of depicted objects.
+
+* The previous architectures upto StyleGan 2 produced great results for static images but failed in case of dynamic images , videos or when details are moved due to texture sticking and lack of .translational as well as rotational equivariance.
+
+<video width="500px" height="300px" controls="controls">
+ <source src="../images/stylegan.mp4" type="video/mp4" /></video>
+ 
+ <video width="500px" height="300px" controls="controls">
+ <source src="../images/left_equivariance.mp4" type="video/mp4"></video>
+
+* The root cause was careless signal processing that causes aliasing in the generator network. 
+
+* Interpreting all signals in the network as continuous, we derive generally applicable, small architectural changes that guarantee that unwanted information cannot leak into the hierarchical synthesis process.
```

Comment:
Kindly convert all First person pronouns to third person
instead of "we" use: "the authors" or "the paper"